doc/source/skips/0-governance.rst

```diff
@@ -0,0 +1,151 @@
+core developer community (including the TC members) fails to reach such a
+consensus in a reasonable timeframe, the TC is the entity to resolve the issue.
+
+Membership of the TC is by nomination by a core developer. A nomination will
+result in discussion that cannot take more than a month and then a vote by
+the core developers that will stay open for a week. TC membership votes are
+subject to a two-third majority of all cast votes as well as a simple majority
+approval of all the current TC members. TC members who do not actively engage
+with the TC duties are expected to resign.
+
+The initial Technical Committee of scikit-image consists of Stéfan van der
+Walt, Juan Nunez-Iglesias, and Emmanuelle Gouillart.
+
+Decision Making Process
+=======================
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the project `mailing list <mailto:scikit-image@python.org>`_
+and the `issue tracker <https://github.com/scikit-image/scikit-image/issues>`_.
+Occasionally, sensitive discussion may occur on a private list.
+
+Decisions should be made in accordance with the :ref:`mission, vision and
+values <values>` of the scikit-image project.
+
+Scikit-image uses a “consensus seeking” process for making decisions. The group
+tries to find a resolution that has no open objections among core developers.
+At any point during the discussion, any core-developer can call for a vote,
+which will conclude one month from the call for the vote. Any vote must be
+backed by a :ref:`scikit-image proposal (SKIP) <skip>`. If no option can gather
+two thirds of the votes cast, the decision is escalated to the TC, which in
+turn will use consensus seeking with the fallback option of a simple majority
+vote if no consensus can be found within a month. This is what we hereafter may
+refer to as “the decision making process”.
+
+Decisions (in addition to adding core developers and TC membership as above)
+are made according to the following rules:
+
+- **Minor documentation changes**, such as typo fixes, or addition / correction of a
+  sentence (but no change of the scikit-image.org landing page or the “about”
+  page), require approval by a core developer *and* no disagreement or requested
+  changes by a core developer on the issue or pull request page (lazy
+  consensus). Core developers are expected to give “reasonable time” to others
+  to give their opinion on the pull request if they’re not confident others
+  would agree.
+
+- **Code changes and major documentation changes** require agreement by *two*
+  core developers *and* no disagreement or requested changes by a core developer
+  on the issue or pull-request page (lazy consensus).
+
+- **Changes to the API principles and changes to dependencies or supported
+  versions** require a :ref:`skip` and follow the decision-making process
```

Comment:
`supported versions` of the library or of the dependencies? If latter is meant, I suggest not to require a SKIP for that.

Reply:
I think this is most likely targeted at **Python** versions. I'll update the language for this.